orchagent/bufferorch.cpp

```diff
@@ -374,6 +428,16 @@ task_process_status BufferOrch::processQueue(Consumer &consumer)
             }
         }
     }
+
+    if (m_ready_list.find(key) != m_ready_list.end())
+    {
+        m_ready_list[key] = true;
```

Comment:
can you please elaborate on it?

Reply:
m_ready_list.find()
m_ready_list[]


---
In reply to: [193597878](https://github.com/Azure/sonic-swss/pull/515#discussion_r193597878) [](ancestors = 193597878)

Reply:
```
if (m_ready_list.find(key) != m_ready_list.end()) 
{ 
    m_ready_list[key] = true;
```
vs
```
auto ready_list_it = m_ready_list.find(key);
if (ready_list_it != m_ready_list.end()) 
{ 
    *ready_list_it = true;
```
I think first case is more readable, although it might be slightly slower in case the compiler will not optimize it.

That code is not on he hot path, so I preferred to use better readability here.